### PR TITLE
Update configuration.asciidoc

### PR DESCRIPTION
**Pull request by leahleahy**

### CODE REVIEW

File: docs/configuration.asciidoc
Comment (leahleahy) [inline docs/configuration.asciidoc:184]: Should we also add the word value here? Boolean value?

File: docs/configuration.asciidoc
Comment (leahleahy) [inline docs/configuration.asciidoc:188]: Deactivated instead of inactivated? Does that still make sense?

File: docs/configuration.asciidoc
Comment (leahleahy) [inline docs/configuration.asciidoc:238]: Is it okay to remove the "in less regexy terms" ?

File: docs/configuration.asciidoc
Comment (leahleahy) [inline docs/configuration.asciidoc:279]: Lines 277-280 didn't really make sense. Could/Should this be rewritten?

File: docs/configuration.asciidoc
Comment (leahleahy) [inline docs/configuration.asciidoc:591]: Should this be rephrased?

File: docs/configuration.asciidoc
Comment (leahleahy) [inline docs/configuration.asciidoc:623]: Is maximum here still correct instead of maximal?

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:28]:
```suggestion
To use this type of setup, which is typical in an ASP.NET Core application, your application's `Startup.cs` file should contain code similar to the following:
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:91]:
```suggestion
In certain scenarios--you won't activate the agent with the `UseElasticApm()` method.
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:92]:
```suggestion
In this case, set the agent log level with <<config-log-level,`ElasticApm:LogLevel`>>, as shown in the following `appsettings.json` file:
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:148]:
```suggestion
Once implemented, you can use the <<config-full-framework-configuration-reader-type, `FullFrameworkConfigurationReaderType`>> setting.
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:238]:
```suggestion
NOTE: The service name must conform to this regular expression: `^[a-zA-Z0-9 _-]+$`. In other words, your service name must only contain characters from the ASCII alphabet, numbers, dashes, underscores, and spaces. Characters in service names that don't match the regular expression will be replaced with the `_` symbol. 
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:279]:
```suggestion
of the deployed revision, e.g. the output of `git rev-parse --verify HEAD`. 
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:354]:
```suggestion
This setting can be changed after the agent starts.
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:383]:
```suggestion
This setting can be changed after agent starts.
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:433]:
```suggestion
If you do find sensitive data in your {es} index, add an additional entry to this list.
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:455]:
```suggestion
Labels are added to all events with the format `key=value[,key=value[,...]]`.
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:502]:
```suggestion
Use this setting if the APM Server requires a secret token, for example, when using our hosted {es} Service on Elastic Cloud.
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:567]:
```suggestion
The path to a PEM-encoded certificate used for SSL/TLS by APM server. Used to perform validation 
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:772]:
```suggestion
This setting can be changed after the agent starts.
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:790]:
```suggestion
Configures the content types to be captured.
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:796]:
```suggestion
This setting can be changed after the agent starts.
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:819]:
```suggestion
NOTE: Setting this to `false` reduces memory allocations, network bandwidth, and disk space used by {es}.
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:842]:
```suggestion
When an incoming HTTP request is detected, its request path will be tested against each element in this list.
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:920]:
```suggestion
A list of namespaces to exclude when reading an exception StackTrace to determine the culprit.
```

File: docs/configuration.asciidoc
Comment (bmorelli25) [inline docs/configuration.asciidoc:941]:
```suggestion
Setting this to `0` disables stack trace collection. Any positive integer value will be used as the maximum number of frames to collect. Setting it to -1 means that all frames will be collected.
```

**Comment by bmorelli25** [comment]
Jenkins, run the tests please

**Comment by bmorelli25** [comment]
@elasticmachine, run elasticsearch-ci/docs

**Review by bmorelli25 — APPROVED** [review]
Pending doc build, LGTM

File: docs/configuration.asciidoc
Comment (russcam) [inline docs/configuration.asciidoc:91]: stray `--` still present- maybe replace with comma?

**Review by russcam — APPROVED** [review]
LGTM, left one small comment

File: docs/configuration.asciidoc
Comment (bmorelli25) [reply docs/configuration.asciidoc:91]: Ahh, dang, I hadn't refreshed the page so this comment didn't show up until after I pressed the merge button. Let me quick open a new PR to get this fixed 🙃

Comment (bmorelli25) [reply docs/configuration.asciidoc:91]: I'm not sure why the "like when you're not using ASP.NET Core" was removed here tbh. I think it makes sense to keep it.

**Pull request merged**
